services/ui-src/src/components/export/ExportedModalDrawerReportSection.tsx

```diff
@@ -18,14 +18,15 @@ export const ExportedModalDrawerReportSection = ({
   const { report } = useContext(ReportContext);
   const { emptyEntityMessage } = exportVerbiage;
   const entities = report?.fieldData?.[entityType];
-  const entityCount = entities?.length;
+  const entityCount = entities ? entities?.length : undefined;
 
   return (
     <Box mt="2rem" data-testid="exportedModalDrawerReportSection">
       <Heading as="h3" sx={sx.dashboardTitle} data-testid="headerCount">
         {verbiage.dashboardTitle} {entityCount}
         {!entityCount && (
           <Text as="span" sx={sx.notAnswered} data-testid="entityMessage">
+            <br />
```

Comment:
Question: What does this do? Would it be better to do it via styling instead of a line break? Line breaks can be finicky on mobile sizes and whatnot.

Reply:
I'm using CSS now, but Design asked for the "not answered" texts to be in a new line.